api-docs/list-clients.md

```diff
@@ -24,9 +24,10 @@ Notes:
       "id": 56,
       "firstName": "Mario",
       "lastName": "Luigi",
+      "fullName": "Mario Luigi",
```

Comment:
To match [get client api](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/api-docs/get-client.md)